dev-example/src/example.test.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  createMockMachineRunner,
+  createMockState,
+  createMockStateOpaque,
+} from '@actyx/machine-runner/lib/test-utils'
+import { Bid, BidderID, Passenger, Requested } from './machines/index.js'
+import { toPrettyJSONString } from './UIMachineCommon.js'
+import { isTaxiRideCancelEnabled } from './UIMachinePassenger.js'
+
+describe('State transformation tests', () => {
+  it('should verify if state transformation as written by the consumer code is correct', () => {
+    const requestDestination = 'destination'
+    const requestPickup = 'pickup'
+    const bidTime = new Date()
+    const bidPrice = 1
+    const bidderId = 'bidderId'
+
+    const machineRunner = createMockMachineRunner(Passenger.Initial, void 0)
+    machineRunner.test.feed([
+      Requested.make({
+        destination: requestDestination,
+        pickup: requestPickup,
+      }),
+      Bid.make({
+        price: bidPrice,
+        time: bidTime.toISOString(),
+      }),
+      BidderID.make({
+        id: bidderId,
+      }),
+    ])
+
+    const auction = machineRunner.get()?.as(Passenger.Auction)
+    expect(auction).toBeTruthy()
+    if (!auction) throw new Error('no auction')
+    const firstBid = auction.payload.bids.at(0)
+    if (!firstBid) throw new Error('no first bid')
+    expect(firstBid.time).toEqual(bidTime)
+    expect(firstBid.price).toEqual(bidPrice)
+    expect(firstBid.bidderID).toEqual(bidderId)
+
+    // Comparing arrays of objects that has a Date member is broken in jest. This doesn't work.
```

Comment:
would it work as `expect(firstBid).toEqual({...})`?

Reply:
It works. Changed per https://github.com/Actyx/machines/pull/30/commits/093563d4b9c378381826866301839e5627e5aca3